code/game/machinery/computer/law.dm

```diff
@@ -6,10 +8,24 @@
 	circuit = /obj/item/circuitboard/aiupload
 	var/mob/living/silicon/ai/current = null
 	var/opened = FALSE
-
 	light_color = LIGHT_COLOR_WHITE
 	light_range_on = 2
-
+	/// sets the cooldown time between uploads when emag'd
+	var/cooldown = 0
+	/// holds the value for when the inherent_laws are counted in countlaws()
+	var/foundlaws = 0
```

Comment:
```suggestion
	var/found_laws = 0
```

snake case please, will need to replace all instances in the code below

Reply:
changes applied